runner/operator/access/v1_0_0/msi/__init__.py

```diff
@@ -0,0 +1,145 @@
+"""""""""""""""""""""""""""""
+" ACCESS-Pipeline MSI workflow operator
+" http://www.github.com/mskcc/access-pipeline/workflows/msi.cwl
+"""""""""""""""""""""""""""""
+
+import os
+import json
+from jinja2 import Template
+
+from runner.models import Port, Run, RunStatus
+from file_system.models import FileMetadata
+from runner.operator.operator import Operator
+from runner.serializers import APIRunCreateSerializer
+from file_system.repository.file_repository import File, FileRepository
+
+
+WORKDIR = os.path.dirname(os.path.abspath(__file__))
+
+class AccessLegacyMSIOperator(Operator):
+
+    # Will be populated after get_sample_inputs()
+    sample_inputs = None
+    number_of_inputs = None
```

Comment:
I know I did it this way at first, but actually Beagle will only call `get_jobs()` so it is better to have `get_jobs()` call `self.get_sample_inputs()` and for these two variables to be returned directly. 

https://github.com/mskcc/beagle/pull/586/files#diff-0abd8d465426319d5adefec646766875f8bf4d6e526c510cb295b1233c60a333R129

Lmk it that makes sense